docs/source/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -332,6 +336,47 @@ mark the last 1-17 people as untracked -- which isn't so much memory
 to drag around. Drawback: this leaves both the number of households
 and the number of simulants as variable (but not that variable).
 
+.. _census_prl_gq_init:
+
+Initializing people living in group quarters
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To initialize approximately N simulants total, including simulants
+residing in Group Quarters when initializing our simulation, we will
+first initialize K individuals in Group Quarters (in three steps,
+described below) and then initialize approximated (N-K) individuals
+into households as described above.
+
+The first step of this process is choosing K; ideally it would be
+sampled from a Binomial distribution, with a probability p_GQ of each
+simulant being in GQ, and p_GQ would itself be sampled from a Beta
+distribution based on the weighted fraction of the population in GQ
+for this geography, with a concentration parameter appropriate to the
+sample size from which the weighted fraction was calculated.  But for
+now, to keep things simple, we will use K = 0.03*N.
+
+Second, to generate K individuals living in group quarters, we will
+use a weighted sample of people in group quarters from the appropriate
+geography from ACS (sampled with replacement, analogously to
+household).  This will provide each simulant residing in GQ with an
+age, sex, race/ethnicity, and geographic location matching the joint
+distribution from ACS.  It does not identify *which* group quarters
+the individual resides in, however, and only provides information on
+whether it is Institutional or Non-institutional GQ (in the TYPE
+variable: 2 = Institutional; 3 = Non-institutional).
+
+The third and final step for initializing GQ simulants is to give each
+a (somewhat inappropriately named) household_id.  Eventually we shall
+accomplish this so that the distribution of GQ sizes match what is
+found in census, but as a simple stand-in for now we will include 6
+special "household_id" values for the six broad types of GQs that we
+wish to represent, and assign simulants to one of the categories
+consistent with their GQ TYPE uniformly at random.  The GQ subtypes of
```

Comment:
when we eventually add complexity to this, some constraints might include
- age-dependent probability of college
- age-dependent probability of nursing home
- (other gq types also prob age-dependent, but these ^ perhaps most obviously so?)
- anyone "retired" unlikely to move into military GQ